sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/ReadmeSamples.java

```diff
@@ -34,12 +34,9 @@
 import java.util.concurrent.atomic.AtomicReference;
 
 /**
- * WARNING: MODIFYING THIS FILE WILL REQUIRE CORRESPONDING UPDATES TO README.md FILE. LINE NUMBERS ARE USED TO EXTRACT
- * APPROPRIATE CODE SEGMENTS FROM THIS FILE. ADD NEW CODE AT THE BOTTOM TO AVOID CHANGING LINE NUMBERS OF EXISTING CODE
- * SAMPLES.
- * <p>
  * Class containing code snippets that will be injected to README.md.
  */
+@SuppressWarnings("unused")
```

Comment:
Why is this needed?

Reply:
It's not, I was just struggling with IntelliJ flagging APIs to be removed for being unused